test/onnx/onnx_import.in.cpp

```diff
@@ -1738,42 +1738,6 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_erf_int32)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_model_hardmax)
```

Comment:
Why should we remove a static test for hardmax?

Reply:
hardmax uses builder::flatten, after removing handling static case dynamic backend is always needed

Reply:
It seems like a regression to me.